spec/factories/facilities.rb

```diff
@@ -1,22 +1,22 @@
 FactoryBot.define do
   factory :facility do
-    transient do
-      fac_group { create(:facility_group) }
-    end
-
     id { SecureRandom.uuid }
-    sequence(:name) { |n| "Facility #{n}" }
-    sequence(:street_address) { |n| "#{n} Gandhi Road" }
-    sequence(:village_or_colony) { |n| "Colony #{n}" }
-    district 'Bathinda'
-    state 'Punjab'
-    country 'India'
-    pin '123456'
-    zone 'Block ABC'
-    facility_type 'PHC'
+    name { "#{facility_type} #{village_or_colony}" + rand(1..100).to_s }
+    street_address { Faker::Address.street_address }
+    village_or_colony { Faker::Address.village }
+    district { Faker::Address.district }
+    state { Faker::Address.state }
+    country { 'India' }
+    pin { Faker::Address.zip_code }
+    zone { Faker::Address.block }
```

Comment:
These changes might really slow down our regular test suite. Could we add a trait for this - eg. `realistic` or `seed`?